manifests/a/Argotronic/ArgusMonitor/7.0.4.2721/Argotronic.ArgusMonitor.installer.yaml

```
@@ -0,0 +1,18 @@
+# Automatically updated by the winget bot at 2024/Jan/08
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: Argotronic.ArgusMonitor
+PackageVersion: 7.0.4.2721
+InstallerLocale: en-US
+MinimumOSVersion: 10.0.0.0
+InstallerType: nullsoft
+UpgradeBehavior: install
+AppsAndFeaturesEntries:
+- ProductCode: ArgusMonitor
+  DisplayVersion: 6.2.3.2667

```

```suggestion
  DisplayVersion: 7.0.4.2721
```
Not sure that this will work, but hopeful.